tests/zipfile/test_unzip.py

```diff
@@ -165,8 +167,18 @@ def test_bad_zip_file(mocker, tmpdir):
         )
 
 
-def test_unzip_url(mocker, tmpdir):
+@pytest.mark.parametrize(
+    'url',
+    [
+        'https://example.com/path/to/fake-repo-tmpl.zip',
+        's3://example-bucket/path/to/fake-repo-tmpl.zip',
+    ],
+)
+@moto.mock_s3
+def test_unzip_url(mocker, tmpdir, url):
     """In `unzip()`, a url will be downloaded and unzipped."""
+    boto3.resource('s3').create_bucket(Bucket='example-bucket')
```

Comment:
I remember from own projects that boto3 testing is something terrible, but it is not clear here what and why it is for https case. 
Please split normal and s3 tests